### PR TITLE
Fix/tm 32

### PR DESCRIPTION
**Pull request by agnealeknavicute**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +0,0 @@
-DATABASE_URL = mongodb+srv://derttor:oN53Ohrjj20B12bM@cluster-task-manager.xobp0bj.mongodb.net/task-manager?retryWrites=true&w=majority
```
Comment (Jevgenijs92) [inline .env:1]: Credentials still can be found in git. We need to re-write git history to remove it completely. Ping me if help is needed on this point.

File: app/seed/route.ts [hunk 2]
```diff
@@ -1,63 +1,93 @@
...
-    const task = await Todo.findOne({ id: id });
-    return JSON.parse(JSON.stringify(task));
-  } catch (e) {
-    console.log(e);
-    return null;
-  }
+    await connectDB();
+    try {
+        const task = await Todo.findOne({ id: id });
+        return JSON.parse(JSON.stringify(task));
```
Comment (jevgenijs-jaunslavietis) [inline app/seed/route.ts:24]: Do we need JSON.parse/stringify there and below?

File: app/types/db-todo-Model.ts
Comment (jevgenijs-jaunslavietis) [inline app/types/db-todo-Model.ts:1]: db-todo-model.ts

File: app/ui/tasks/card-delete-component.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+'use client';
+import { MenuItem } from '@chakra-ui/react';
+import React from 'react';
+import { useRouter } from 'next/navigation';
+import { deleteTask } from '@/app/seed/route';
+
+interface ICardDeleteComponent {
```
Comment (jevgenijs-jaunslavietis) [inline app/ui/tasks/card-delete-component.tsx]: There is no need to add `I` prefix for interfaces. It is obvious that it is interface if we check where it is defined 😄

Comment (jevgenijs-jaunslavietis) [reply app/ui/tasks/card-delete-component.tsx]: Usually, we call such interfaces something like `CardDeleteComponentProps`

**Pull request merged**